tests/antithesis/docker-compose.yml

```diff
@@ -21,7 +21,7 @@ services:
         chown -R ${USER_ID:-root}:${GROUP_ID:-root} /var/etcddata0 /var/etcddata1 /var/etcddata2 /var/report
 
   etcd0:
-    image: 'etcd-server:latest'
+    image: 'etcd-server:${VERSION:-latest}'
```

Comment:
Don't think we should customize the image name as in antithesis integration we need to pass a parameter with list of all images. Having dynamic image names makes it harder to ensure parameter is correctly passed.